### ISSUE TITLE
Adding a reviewer doesn't update list

### ISSUE THREAD

**Issue by digitarald** [microsoft/vscode-pull-request-github#4405]
Type: <b>Bug</b>

- Added a reviewer in a PR
- List stays empty
- Opening the quick pick again, reviewer is already marked added

Extension version: 0.57.2023011709
VS Code version: Code - Insiders 1.75.0-insider (1379f0375440df946adc62f7afb8375dd45d5adf, 2023-01-17T05:21:02.509Z)
OS version: Darwin x64 22.2.0
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|4, 5, 5|
|Memory (System)|64.00GB (2.39GB free)|
|Process Argv|--enable-proposed-api donjayamanne.kusto --log trace --crash-reporter-id 4cf0a6f0-aaf8-4da3-9b2b-81f6c639c12b|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by alexr00** [comment]
> List stays empty

I can't reproduce this. Is there a specific repo that this is reproducing in?

**Comment by digitarald** [comment]
No, sorry. I saw it once yesterday and also didn't see any errors in the logs.
